api_validation/pom.xml

```diff
@@ -72,8 +72,7 @@
         <dependency>
             <groupId>ai.rapids</groupId>
             <artifactId>cudf</artifactId>
-            <classifier>${cuda.version}</classifier>
```

Comment:
It's unclear whether we're going to be able to remove the classifier going forward.  When another major CUDA version comes along, there could be a new GPU architecture that comes along with it, and there's no guarantee that running on the later major version will work on a machine with an earlier major CUDA version.  Therefore we may always have a classifier for at least the major CUDA versions.

I suggest leaving the classifier in-place for now, we can remove it in a followup if indeed we know that the classifier will be going away for all future cudf versions.

Reply:
make sense. Let me bring the classifier back